docs/codeql/writing-codeql-queries/creating-path-queries.rst

```diff
@@ -30,7 +30,7 @@ For more language-specific information on analyzing data flow, see:
 - ":ref:`Analyzing data flow in JavaScript/TypeScript <analyzing-data-flow-in-javascript-and-typescript>`"
 - ":ref:`Analyzing data flow in Python <analyzing-data-flow-in-python>`"
 - ":ref:`Analyzing data flow in Ruby <analyzing-data-flow-in-ruby>`"
-
+- ":ref:`Analyzing data flow in Swift <analyzing-data-flow-in-swift>`"
 
 Path query examples
```

Comment:
@geoffw0 - I'm wondering if Swift needs to be added to the list of languages that need you to explicitly import the data flow library or not. See: [Constructing a path query](https://codeql.github.com/docs/writing-codeql-queries/creating-path-queries/#constructing-a-path-query).

Reply:
Yes, it appears the dataflow library does need to be imported in Swift (for both old and new `ConfigSig` style dataflow queries).  It doesn't like:
```
import semmle.code.swift.dataflow.DataFlow
import DataFlow::PathGraph
```
though, it wants:
```
import codeql.swift.dataflow.DataFlow
import DataFlow::PathGraph
```
(I guess this difference is covered by the phrasing "such as")

Reply:
Updated.